src/test/kotlin/org/rust/lang/core/completion/RsCompletionTestFixtureBase.kt

```diff
@@ -0,0 +1,125 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.lang.core.completion
+
+import com.intellij.codeInsight.lookup.LookupElement
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFileFilter
+import com.intellij.psi.impl.PsiManagerEx
+import com.intellij.testFramework.fixtures.CodeInsightTestFixture
+import com.intellij.testFramework.fixtures.impl.BaseFixture
+import org.rust.*
+
+abstract class RsCompletionTestFixtureBase<IN>(
+    protected val myFixture: CodeInsightTestFixture
+) : BaseFixture() {
+
+    private val project: Project get() = myFixture.project
+
+    fun executeSoloCompletion() {
+        val variants = myFixture.completeBasic()
```

Comment:
Maybe rename `variants` to `lookups` (here and in other functions) for consistency with `checkContainsCompletion`